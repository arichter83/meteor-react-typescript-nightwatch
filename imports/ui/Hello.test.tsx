import * as React from 'react'
import Hello from './Hello'
import renderer from 'react-test-renderer'

test('<Hello /> counter increases', () => {
  const component = renderer.create(<Hello />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

  // trigger button click
  tree.children[0].props.onClick()
  expect(component.toJSON()).toMatchSnapshot();

});
