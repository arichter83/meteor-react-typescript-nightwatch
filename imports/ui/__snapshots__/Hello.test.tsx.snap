// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hello /> counter increases 1`] = `
<div>
  <button
    onClick={[Function]}
  >
    Click Me
  </button>
  <p>
    You've pressed the button 
    0
     times.
  </p>
</div>
`;

exports[`<Hello /> counter increases 2`] = `
<div>
  <button
    onClick={[Function]}
  >
    Click Me
  </button>
  <p>
    You've pressed the button 
    1
     times.
  </p>
</div>
`;
